x-pack/plugins/apm/server/lib/anomaly_detection/create_anomaly_detection_jobs.ts

```diff
@@ -47,7 +56,7 @@ export async function createAnomalyDetectionJobs(
     const dataViewName = indices.metric;
```

Comment:
Not related to this PR but can you please rename `dataViewName` to `apmMetricIndex`? 

```suggestion
    const apmMetricIndex = indices.metric;
```